torch/onnx/symbolic_opset9.py

```diff
@@ -362,6 +362,18 @@ def atan(g, self):
     return g.op("Atan", self)
 
 
+def atan2(g, input, other):
```

Comment:
It's not clear to me how to get from the definition [here](https://en.wikipedia.org/wiki/Atan2#Definition_and_computation) to this function body.
Please add a comment linking to a reference that explains the calculation.